src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -53,6 +53,10 @@ public void setTestVerbosity(){
     public static final String publicTestDir = new File(gatkDirectory, publicTestDirRelative).getAbsolutePath() + "/";
     public static final String publicTestDirRoot = publicTestDir.replace(publicTestDirRelative, "");
 
+    public static final String moduleTestDataDir = publicTestDir + "org/broadinstitute/hellbender/";
```

Comment:
The term "module" is a bit confusing here -- maybe `packageRootTestDir`, since `org.broadinstitute.hellbender` is our root Java package?

Also change `DataDir` to just `Dir` in the names of these constants.